nbstripout/_nbstripout.py

```diff
@@ -504,7 +526,11 @@ def main():
 
     if not args.files and input_stream:
         try:
-            process_notebook(input_stream, output_stream, args, extra_keys)
+            any_local_change = process_notebook(input_stream, output_stream, args, extra_keys)
+            any_change = any_change or any_local_change
```

Comment:
```suggestion
            if process_notebook(input_stream, output_stream, args, extra_keys):
                any_change = True
```
Wouldn't this be easier?

Reply:
Done